src/library/Cardano/Fee.hs

```diff
@@ -128,21 +126,47 @@ newtype FeeEstimator i o = FeeEstimator
 data FeeOptions i o = FeeOptions
     { feeEstimator
         :: FeeEstimator i o
+        -- ^ Estimate fees based on selected inputs and requested outputs.
+
     , dustThreshold
         :: DustThreshold
+        -- ^ Change addresses below the given threshold will be evicted
+        -- from the created transaction. Setting 'dustThreshold' to 0
+        -- removes output equal to 0
+
+    , onDanglingChange
+        :: OnDanglingChange
+        -- ^ What do to when we encouter a dangling change output.
+        -- See 'OnDanglingChange'
     } deriving Generic
 
+-- | We call 'dangling' a change output that would be too expensive to add. This
+-- can happen when a change output is small, but adding it generate more fees
+-- than not having it.
```

Comment:
`dust` refers to small UTxO, but the sense given to `small` is completely arbitrary and set by users (for instance, in the Migration algorithm we decide that anything smaller than the constant cost of transaction is dust). 

a `dangling` output however is "well-defined". It is an output that costs more to add than to not have; because adding increases the fee beyond the output value itself. In that sense, I think that you currently think as `dust` is actually `dangling` :)